packages/common/src/components/nu-theme/index.js

```diff
@@ -0,0 +1,36 @@
+// new theme based on redesign
+// this follows the System UI Theme Specification
+// Ref: https://system-ui.com/theme
+import components from './components';
+
+const colors = {
```

Comment:
Maybe we can have a separate file where we got all the colors, ... and then 2 theme files (1 for `style-components` and 1 for `react-ui`), so the colors stay in sync in both theming systems? 🤔